changelog.md

```diff
@@ -1,7 +1,17 @@
 # Changelog
 
+## 4th June 2019 19:00UTC
+
+* Extended [Add key cutter command](integrations.md#add-key-cutter-command) parameters with `KeyMode`.
+* Extended [Key cutter command data](integrations.md#key-cutter-command-data) response with `KeyMode`.
+
+## 21th May 2019 19:00UTC
+
+* Added [Integration event webhook](guidelines.md#Integration-event-webhook) to [Guidelines](guidelines.md).
```

Comment:
Introduce new section for webhooks.

Reply:
That makes sense when we have webhooks as a way to get data from Mews. This webhook is about an integrator getting alerts on their integration status, it's related to guidelines